docs/modules/servers/pages/15-minute-demo.adoc

```diff
@@ -96,28 +80,28 @@ Create a new "test.local" domain:
 
 [source,bash]
 ----
-./james addDomain test.local
+docker exec james james-cli addDomain test.local
 ----
 
 List the domains again to ensure that "test.local" has successfully been added:
 
 [source,bash]
 ----
-./james listDomains
+docker exec james james-clivv listDomains
 ----
 
 Add the user "testuser" to the "test.local" domain with password "password":
 
 [source,bash]
 ----
-./james addUser testuser@test.local password
+docker exec james james-cliv addUser testuser@test.local password
```

Comment:
cliv here

Reply:
I have a sensibility issue on my keyboard, it is very annoying...